### ISSUE TITLE
Tigger for cron jobs doesn't actually start one

### ISSUE THREAD

**Issue by trhumphries** [vmware-tanzu/octant#1323]
**What steps did you take and what happened:**
I click the trigger menu item on a cron job, the alert that the cron job was started show at the top of the browser, however no job was actually started

**What did you expect to happen:**
I expected the job to be started
**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

**Environment:**

- Octant version 0.15.0
- Kubernetes version 1.18.2
- OS(macOS 10.15

**Comment by GuessWhoSamFoo** [comment]
Are you able to create a job via `kubectl create job --from=cronjob/<cron-name> <job-name>`? Do events say anything on a manual trigger?

**Comment by trhumphries** [comment]
Yes I can create it via kubectl, and there is nothing in the Events for the manga; trigger.

Terry Humphries
STRATUS Operations Lead
GTP Services, LLC
816-377-1113

[cid:image001.png@01D3FC1B.2842F670]

This email is the property of GTP Services, LLC and may contain confidential and/or privileged information.
If you are not the intended recipient or have received this e-mail in error please notify the sender immediately and delete this e-mail.
Any unauthorized copying, disclosure or distribution of the material in this e-mail is strictly forbidden.



On Sep 4, 2020, at 2:18 PM, Sam Foo <notifications@github.com<mailto:notifications@github.com>> wrote:



Are you able to create a job via kubectl create job --from=cronjob/<cron-name> <job-name>? Do events say anything on a manual trigger?

**Comment by wwitzel3** [comment]
I'm unable to replicate this with examples from https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/
@trhumphries are you able to provide the YAML for the cronjob with any sensitive details scrubbed.

Maybe it is something with how Octant is interacting with the specific way our cronjob is declared. It's a long shot, but worth checking to be sure.

**Comment by bazron** [comment]
I have the same issue. The banner on top tells me the that the job with name xxx has started but in fact nothing starts. I'm able to trigger the job through kubectl (and the kubernetes-dashboard). Nothing in Events.

Environment:

Octant version 0.19.0
Kubernetes version: 1.17.12 on EKS
OS(Windows 10 pro): 10..0.19042
